ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala

```diff
@@ -253,8 +254,10 @@ object BigSampler extends Command {
       )
       // Prompts FileSystems to load service classes, otherwise fetching schema from non-local fails
       FileSystems.setDefaultPipelineOptions(opts)
+      val fileNames = FileSystems.`match`(input).metadata().asScala.map(_.resourceId().getFilename)
```

Comment:
This will incur a network call in cloud storage cases right? Do failures here produce meaningful errors?